Makefile

```diff
@@ -9,13 +9,13 @@ curl -o $@ https://github.com/clojure-emacs/clojuredocs-export-edn/raw/master/ex
 
 # .EXPORT_ALL_VARIABLES passes TEST_PROFILES to Lein so that it can inspect the active profiles, which is needed for a complete Eastwood setup:
 test: clean .EXPORT_ALL_VARIABLES
-	lein with-profile -user,-dev,+$(VERSION),$(TEST_PROFILES) test
+	lein with-profile -user,-dev,+$(VERSION),$(TEST_PROFILES),-provided test
```

Comment:
My thinking is that we should exercise the test suite with both `-provided` and no `-provided`, ensuring that the functionality works under any scenario.

Are you on board with the idea?

Reply:
Yeah, that makes sense.